src/pages/workspaces/workspace/analysis/modals/DeleteDiskChoices.ts

```diff
@@ -56,18 +68,29 @@ export const DeleteDiskChoices = ({ existingEnvironmentConfig, deleteDiskSelecte
         ])
       ]),
       Utils.cond(
-        [runtime.tool === 'RStudio', () => h(SaveFilesHelpRStudio)],
-        [runtime.cloudService === cloudServices.GCE, h(() => SaveFilesHelp(false))],
-        () => h(SaveFilesHelpAzure)
+        [toolLabel === 'RStudio', () => h(SaveFilesHelpRStudio)],
+        [cloudService === cloudServices.GCE, () => SaveFilesHelp(false)],
+        [Utils.DEFAULT, () => h(SaveFilesHelpAzure)]
       )
     ])
   )
 }
 
 
-export const DeleteEnvironment = ({ id, existingEnvironmentConfig, deleteDiskSelected, setDeleteDiskSelected, setViewMode, renderActionButton, hideCloseButton, onDismiss }:
-{ id:string; existingEnvironmentConfig:any; deleteDiskSelected:boolean; setDeleteDiskSelected:(p1:boolean)=>void; setViewMode: (value: React.SetStateAction<string|undefined>) => void; renderActionButton: ()=> React.ReactElement<any, any>; hideCloseButton:boolean; onDismiss: React.MouseEventHandler<Element>}) => {
-  const { runtime, persistentDisk } = existingEnvironmentConfig
+export const DeleteEnvironment = ({ id, runtimeConfig, persistentDisk, deleteDiskSelected, setDeleteDiskSelected, setViewMode, renderActionButton, hideCloseButton, onDismiss, toolLabel, computeRegion }:
+{
+  id:string
+  runtimeConfig?:RuntimeConfig
+  persistentDisk?:any//TODO: retype this as PersistentDisk
+  deleteDiskSelected:boolean
+  setDeleteDiskSelected:(p1:boolean)=>void
+  setViewMode: (value: React.SetStateAction<string|undefined>) => void
+  renderActionButton: ()=> React.ReactElement<any, any>
+  hideCloseButton:boolean
+  onDismiss: React.MouseEventHandler<Element>
+  toolLabel:string
+  computeRegion?:string
+}) => {
```

Comment:
This right here is a shining example of why line length, linting, and styling would be useful to enforce.

Reply:
Follow up - Seems like it's due to being TypeScript specific syntax.